ProcessMaker/Repositories/CaseRepository.php

```diff
@@ -21,34 +26,27 @@ public function __construct(protected CaseParticipatedRepository $caseParticipat
      */
     public function create(ExecutionInstanceInterface $instance): void
     {
-        if (is_null($instance->case_number) || $this->checkIfCaseStartedExist($instance->case_number)) {
+        if (is_null($instance->case_number)) {
+            \Log::error('Case number is required. instance: ' . $instance->getKey());
```

Comment:
How this scenario could happen, could you add some steps or a test to replicate it?

Reply:
This scenario was given in this test `UploadPublicFileTest`, since the `processRequest` is instantiated but not stored